pkg/k8s/network_policy_v1beta.go

```diff
@@ -139,6 +148,16 @@ func ParseNetworkPolicyV1beta1(np *v1beta1.NetworkPolicy) (api.Rules, error) {
 		egresses = append(egresses, egress)
 	}
 
+	// Isolate namespace on ingress
+	if len(ingresses) == 0 && isV1beta1PolicyType(np.Spec.PolicyTypes, v1beta1.PolicyTypeIngress) {
```

Comment:
Was `PolicyType` ever part of beta? I thought it was introduced after GA? I'm confused that these fields are even specified.

Reply:
I was surprised about this too, but it is indeed part of beta.